extra-utils/slurp/autobuild/defines

```diff
@@ -1,4 +1,4 @@
 PKGNAME=slurp
 PKGSEC=utils
-PKGDEP="wayland wayland-protocols cairo libxkbcommon"
-PKGDES=" Select a region in a Wayland compositor"
+PKGDEP="wayland wayland-protocols cairo libxkbcommon scdoc"
+PKGDES="Select a region in a Wayland compositor"
```

Comment:
What does this even mean?

Reply:
slurp is usually used alongside grim. slurp allows the user to select a region and pass the starting coordinate and size to grim, so that grim can save the region as an image.

Here's an example.
```bash
# Select a region, take a screenshot and save it to the clipboard.
grim -g "$(slurp)" - | wl-copy
```